GruntTasks/Options/concat.orchestrajs.js

```diff
@@ -89,15 +89,19 @@ module.exports = {
         'web/built/openorchestrabackoffice/js/tab/TabElementFormView.js',
         'web/built/openorchestrabackoffice/js/tab/tabViewFormLoader.js',
 
+        //--[ DATATABLE ]--//
+        'web/built/openorchestrabackoffice/js/dataTable/plugins/InputFullPagination.js',
```

Comment:
Is it possible that all the files in the dataTable folder can be loaded without regarding their order ?